ibm/service/power/ibm_pi_constants.go

```diff
@@ -4,21 +4,29 @@ import "time"
 
 const (
 
-	// Keys
-	PIKeys    = "keys"
-	PIKeyName = "name"
-	PIKey     = "ssh_key"
-	PIKeyDate = "creation_date"
+	// -- Capture -------------------------------------------------------------
 
-	// SAP Profile
-	PISAPProfiles         = "profiles"
-	PISAPProfileCertified = "certified"
-	PISAPProfileCores     = "cores"
-	PISAPProfileMemory    = "memory"
-	PISAPProfileID        = "profile_id"
-	PISAPProfileType      = "type"
+	// -- Cloud Connection ----------------------------------------------------
+
+	// -- Cloud Instance ------------------------------------------------------
+	// Required Arguments
+	Arg_CloudInstanceID = "pi_cloud_instance_id"
```

Comment:
Some are following with Arg_XXX Att_XX for repsective argument/attributes but some are like PISAPProfiles...
can we have some common standard to represent